py/specs/outer-joins.toml

```diff
@@ -10,20 +38,42 @@ statement_files = ["sql/employees.sql", "sql/offices.sql"]
         source = "data/offices.json"
 
 statements = [
-    "refresh table employees, offices"
+    "refresh table actor, film_actor, film, film_category, category, employees, offices"
 ]
 
 [[queries]]
-statement = "select e.name as employee, o.name as office from employees e left join offices o on e.office_id = e.id"
-iterations = 250
+statement = """SELECT 
```

Comment:
Why do we need such complex query here?
Shouldn't we at least keep a simple left outer join?